ddl/db_integration_test.go

```diff
@@ -2927,3 +2927,79 @@ func (s *testIntegrationSuite3) TestAvoidCreateViewOnLocalTemporaryTable(c *C) {
 	tk.MustExec("create view vv as select * from v0")
 	checkCreateView()
 }
+
+func (s *testIntegrationSuite3) TestDropTemporaryTable(c *C) {
```

Comment:
Try this: drop table normal_table, temp_table;
Where the normal_table doesn't exist, which leads to rollback.
I assume it's both OK that `temp_table` exists or not.